ios/brave-ios/Sources/Brave/Frontend/Brave Translate/BraveTranslateSession.swift

```diff
@@ -8,6 +8,12 @@ import SwiftUI
 import Translation
 import os.log
 
+// If set to true, Apple Translate will be used
+// If set to false, Brave Translate will be used
+// Apple Translate is causing a lot of issues with an unknown reason.
+// IE: It won't translate French to English, etc... so it's disabled atm.
+private let useAppleTranslate = false
```

Comment:
We should open an issue to re-enable Apple Translate & add to this comment. Not sure if this needs mentioned to privacy / security as initial review expected Apple Translate for iOS 18+?

Reply:
With feature flag for Apple Translate enabled, Apple translate is working as expected for me again 🎉 . Seems to have broken in https://github.com/brave/brave-core/pull/27892.